src/lib/apiTypesV2/BatchQuery.cpp

```diff
@@ -53,38 +53,6 @@ BatchQuery::~BatchQuery()
 
 /* ****************************************************************************
 *
-* BatchQuery::check - 
-*/
-std::string BatchQuery::check(ConnectionInfo* ciP, RequestType requestType)
```

Comment:
It was probably never used, which is a bit funny.
The check methods form a tree and a father calls the check methods of its 'sons'.
This will not change.
Perhaps/Probably all the fixes would be covered by simply calling the check method ...

Is there any check whatsoever on scopes, in the current code?

For background, in v1 parsing, all check methods are called after parsing the payload, in one unique place in the source code. V2 parsing has no unique point and I believe we've lost the use of the check methods.
We should try to get that very important mechanism for v2 as well.

I would not remove these methods but instead try to call them, after parsing the incoming payload.